### ISSUE TITLE
Error on database creation 

### ISSUE THREAD

**Issue by gabbianiiii** [lbighetti/hades#55]
I got error when do the command “mix ecto.create”.

I need to do anything before run this command? 

P.S.: I using Postgres as database

**Comment by lbighetti** [comment]
@pacheco7 can you paste the error here ?

**Comment by gabbianiiii** [comment]
Sure!

Here's the error:
```
** (RuntimeError) :database is nil in repository configuration
    lib/ecto/adapters/postgres.ex:110: Ecto.Adapters.Postgres.storage_up/1
    lib/mix/tasks/ecto.create.ex:40: anonymous fn/3 in Mix.Tasks.Ecto.Create.run/1
    (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:737: Enum.each/2
    (mix) lib/mix/task.ex:314: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:80: Mix.CLI.run_task/2
    (elixir) lib/code.ex:678: Code.require_file/2
Migrating the data...
[error] GenServer #PID<0.216.0> terminating
** (KeyError) key :hostname not found in: [types: Ecto.Adapters.Postgres.TypeModule, port: 5432, name: Hades.Repo.Pool, otp_app: :hades, repo: Hades.Repo, timeout: 15000, pool_timeout: 5000, adapter: Ecto.Adapters.Postgres, pool_size: 1, pool: DBConnection.Poolboy]
    (elixir) lib/keyword.ex:377: Keyword.fetch!/2
    (postgrex) lib/postgrex/protocol.ex:54: Postgrex.Protocol.connect/1
    (db_connection) lib/db_connection/connection.ex:134: DBConnection.Connection.connect/2
    (connection) lib/connection.ex:622: Connection.enter_connect/5
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Last message: nil
State: Postgrex.Protocol
** (exit) exited in: :gen_server.call(#PID<0.444.0>, {:checkout, #Reference<0.3566956064.3894411267.215600>, true, :infinity}, 5000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
    (db_connection) lib/db_connection/poolboy.ex:112: DBConnection.Poolboy.checkout/3
    (db_connection) lib/db_connection.ex:920: DBConnection.checkout/2
    (db_connection) lib/db_connection.ex:742: DBConnection.run/3
    (db_connection) lib/db_connection.ex:1133: DBConnection.run_meter/3
    (db_connection) lib/db_connection.ex:584: DBConnection.prepare_execute/4
    (ecto) lib/ecto/adapters/postgres/connection.ex:86: Ecto.Adapters.Postgres.Connection.execute/4
    (ecto) lib/ecto/adapters/sql.ex:256: Ecto.Adapters.SQL.sql_call/6
    (ecto) lib/ecto/adapters/sql.ex:198: Ecto.Adapters.SQL.query!/5
    (ecto) lib/ecto/adapters/postgres.ex:85: anonymous fn/4 in Ecto.Adapters.Postgres.execute_ddl/3
    (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3
    (ecto) lib/ecto/adapters/postgres.ex:85: Ecto.Adapters.Postgres.execute_ddl/3
    (ecto) lib/ecto/migrator.ex:44: anonymous fn/2 in Ecto.Migrator.migrated_versions/2
    (ecto) lib/ecto/migrator.ex:300: Ecto.Migrator.verbose_schema_migration/3
    (ecto) lib/ecto/migrator.ex:157: Ecto.Migrator.run/4
    (ecto) lib/mix/tasks/ecto.migrate.ex:83: anonymous fn/4 in Mix.Tasks.Ecto.Migrate.run/2
    (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:737: Enum.each/2
    (mix) lib/mix/task.ex:314: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:80: Mix.CLI.run_task/2
  ```

**Comment by rafaels88** [comment]
Probably because you haven't set your database `ENV`s? In this case, `DATA_DB_USER`, `DATA_DB_PASS` and `DATA_DB_HOST`. (The error above is complaining about this last one, but you need all three).

Ref.: https://github.com/lbighetti/hades/blob/51f587ece71ff924bb59b8ed55c80b9193933903/config/dev.exs#L44

**Comment by lbighetti** [comment]
Just made the PR #57, adds default values in the dev.exs to make this setup smoother. 
A review there would be welcome @rafaels88 @pacheco7 :)

**Comment by lbighetti** [comment]
@pacheco7 I just merged in #57  (thanks for review @rafaels88 !)
If you try now it should work with postgres default configs (postgres/postgres@localhost)

Please try the lastest and let us know how that goes!

**Comment by gabbianiiii** [comment]
Solved by #57

**Comment by lbighetti** [comment]
great!
